docs/content/docs/KnownIssues.md

```diff
@@ -11,3 +11,14 @@ type: docs
 Disable **Settings > Experimental > Only sync the active variant**
 
 https://stackoverflow.com/questions/54201216/android-studio-3-3-new-gradle-sync-is-not-supported-due-to-containing-kotlin-mo/
+
+## Tests run from Android Studio are failed in runtime with `java.lang.NoClassDefFoundError`
+
+The problem occurs only when running a test from Android Studio because of wrongly generated Run Configuration.
+Wherein test run from command line via `./gradlew test` ends up normally.  
+Mostly this reproduced in tests that use Kotlin extensions from Test Fixtures from other Gradle modules.
```

Comment:
```suggestion
When running tests from Android Studio you could get `java.lang.NoClassDefFoundError`. It happens because Android Studio generated wrong Run Configuration. \
You can check it by run tests from command line `./gradlew test`. If command line run ended normally you had a problem.
Mostly this reproduced in tests that use Kotlin extensions from Test Fixtures from other Gradle modules.
```

Reply:
I've re-written the first sentence according to https://github.com/avito-tech/avito-android/pull/234#discussion_r389233392. But I don't understand your suggestion about the second sentence.